specs/NavigationViewTemplateSettings-OpenPaneLength.md

```diff
@@ -0,0 +1,94 @@
+NavigationViewTemplateSettings.OpenPaneLength
```

Comment:
For the most part, I think this is a pretty good example draft. It contains most everything we'd need to pop a corresponding API ref topic. Is it possible to review the raw template? There are a few small issues in this draft that don't make it directly portable to a ref doc. This is fine if PMs hand a spec off to writers to pop the topics as we can pull the relevant details and fit them in to the correct sections of the topic (for example, the property description specifies the default, which would typically be specified in the property type description). It's important to keep the main description as concise as possible due to it being consumed for index topics (class/namespace/etc) and intellisense.

Reply:
This particular example was used to directly pop a ref topic, which then needed the refinements described. So some duplication of effort (create spec, pop ref topic with info in spec, writer review comments suggesting the necessary revisions, revisions completed, re-review, etc).

Reply:
> Is it possible to review the raw template

Yes! Let's open a separate Issue (or PR) on https://github.com/microsoft/WindowsAppSDK/blob/main/specs/spec_template.md.